components/camel-cdi/src/main/java/org/apache/camel/cdi/DelegateProducer.java

```diff
@@ -44,6 +44,10 @@ public void dispose(T instance) {
         return delegate.getInjectionPoints();
     }
 
+    public Producer<T> getDelegate() {
```

Comment:
The method visibility can be package private, or the field can be declared protected and the method removed.